client/lib/plans/constants.js

```diff
@@ -38,6 +38,7 @@ export const FEATURE_CUSTOM_DOMAIN = 'custom-domain';
 export const FEATURE_FREE_THEMES = 'free-themes';
 export const FEATURE_UNLIMITED_PREMIUM_THEMES = 'premium-themes';
 export const FEATURE_3GB_STORAGE = '3gb-storage';
```

Comment:
Do we need `FEATURE_3GB_STORAGE` any more? Either we remove this or actually change the description of `FEATURE_3GB_STORAGE` instead of creating a new one. What do you think? :)